static/app/utils/feedback/item/types.tsx

```diff
@@ -10,7 +10,7 @@ export interface FeedbackItemResponse {
     name: null | string;
     version: null | string;
   };
-  contact_email: string;
+  contact_email: null | string;
```

Comment:
since we made `contact_email` optional in the backend, i assumed `null` was a possible option - there's a part in the blueprint where we forgot to update the fact that it's optional, which is what i was trying to fix in my blueprint PR

Reply:
kk, i'm not keeping up enough obvs right now!

Reply:
Yeah, it could be possible